web/src/components/onboarding/ReturnToPreviousBusinessBar.tsx

```diff
@@ -20,7 +21,10 @@ export const ReturnToPreviousBusinessBar = (props: Props): ReactElement | null =
   const { updateQueue, userData } = useUserData();
   const { state } = useContext(AuthContext);
   const router = useRouter();
-  const businessName = getNavBarBusinessTitle(props.previousBusiness, state.isAuthenticated);
+  const businessName = getNavBarBusinessTitle(
+    props.previousBusiness,
+    state.isAuthenticated === IsAuthenticated.TRUE
```

Comment:
This was just a much needed simplification 

I still think this could be refactored to be simpler as many of the use cases are better divided now because of how I chose to do our cases for, onboarding, landing, guest and authed nav,  but its actuallly just complicated enough to probably warrent a follow up instead